File: Tugas coding modul 6/coding c/PRAK602-2310817120005-Noviana_Nur_Aisyah.c
#include <stdio.h>

void zetsu_putih(int jumlah, int zetsu[jumlah]) {
    for (int i = 0; i < jumlah; i++) {
        printf("%d ", zetsu[i] * (i + 1));
    }
}

int main() {
    int bebas;

    scanf("%d", &bebas);

    int zetsu[bebas];
    for (int i = 0; i < bebas; i++) {
        scanf("%d", &zetsu[i]);
    }

    printf("\n");
    zetsu_putih(bebas, zetsu);

    return 0;
}

File: Tugas coding modul 6/coding py/PRAK602-2310817120005-Noviana_Nur_Aisyah.py
def zetsu_putih(jumlah, zetsu):
    for i in range(jumlah):
        print(zetsu[i] * (i + 1), end=" ")

bebas = int(input())

zetsu = [int(j) for j in input().split()]

print()
zetsu_putih(bebas, zetsu)